### ISSUE TITLE
A few questions regarding basic configuration

### ISSUE THREAD

**Issue by HunterretnuH** [ipod825/ranger.nvim#1]
Hello,
I have problem setting this plugin properly and there is not much information available. Can you help?
Questions:
1. Is it possible and how to configure this plugin to behave like NERDTreeToggle. Details: 
    *  set one key to toggle range.nvim on/off;
    *  don't open preview on the right and leave current buffer not changed.
2. If I i.e. use "S" shortcut to sort the menu is narrow even when there is a lot of space available, why? Is it bug or requires additional config?
3. Is the only way to cancel i.e. sort action ("S" shortcut) to close window with :q? In my opinion Escape should close these windows.
4. How to enable tree mode? It's visible on screenshots, but I haven't found keyword "tree" in whole repository and no information related to this mode.
5. It would be good to add information to readme about installing fonts to have icons as I had spent quite some time getting it to work.
6. Suggestion: Add information to readme how to install using other popular plugin managers. I used this (VimPlug) with this simple configuration to get it to work:
  ```
  Plug 'ipod825/libp.nvim'
  Plug 'ipod825/ranger.nvim'
  Plug 'preservim/nerdtree'
  lua << EOF
  require("ranger").setup({
      hijack_netrw = true,
  })
  EOF
  ```
Some questions are quite basic, but I don't have enough time available to spend multiple hours trying to tweak it only to realize that this plugin is not for me and there isn't enough information available to make a well informed decision. Default configuration (or lack of it) is completely not intuitive for me and above questions would allow me to have basic working configuration.

**Comment by ipod825** [comment]
1.    Is it possible and how to configure this plugin to behave like NERDTreeToggle. Details:
        set one key to toggle range.nvim on/off;
        don't open preview on the right and leave current buffer not changed.

I could implement the functionality to toggle ranger while maintaining the current file.

2.    If I i.e. use "S" shortcut to sort the menu is narrow even when there is a lot of space available, why? Is it bug or requires additional config?

There's a known issue with trailing white space. It happens when user have `set number` or `set relativenumber` by default. I'll try to fix that. However, I am not sure why you only encounter these problem when sorting the content. More details is needed here.


3.    Is the only way to cancel i.e. sort action ("S" shortcut) to close window with :q? In my opinion Escape should close these windows.

The window UI is powered by libp. I can add filetype to all libp window and have user define custom mapping on FileType event. As you said, it's "you opinion" that Escape should close these windows, so these should be user-customizable instead of built-ins.


4.    How to enable tree mode? It's visible on screenshots, but I haven't found keyword "tree" in whole repository and no information related to this mode.

Default mapping is `za` -> action.toggle_expand.

5.    It would be good to add information to readme about installing fonts to have icons as I had spent quite some time getting it to work.

Sorry about that. It's nerdfont and is used by a lot of plugins. Haven't thought about this as I've been using it for a long time. But yes, it's painful to figure out how to set it up.

6.    Suggestion: Add information to readme how to install using other popular plugin managers. I used this (VimPlug) with this simple configuration to get it to work:

Sure.
